### DISCUSSION TITLE
ERT parametric domain depth

### DISCUSSION THREAD

**Discussion by anonymouscapibara** [gimli-org/pyGIMLi#697] [Q&A]
Hi all,

I am trying to understand how PyGimli automatically sets the depth of the parametric domain (the parameter mesh for the inversion).
Referring to an old paper by Gunther et al. (2006 - [https://doi.org/10.1111/j.1365-246X.2006.03011.x](url)), I read: 

> The parameter mesh describes the cells, whose resistivities are to be determined and includes the topography of the target area. The lateral model extension is determined by the electrode layout plus a surrounding region in the dimension of several electrode spacings. To determine the model depth we extend the ideas of Roy and Apparao (1971) and Barker (1989) to multielectrode data. A 1-D-sensitivity function for a homogeneous half-space is calculated and summed up for all data. A cumulative value of 90 per cent depicts a reasonable model depth and can be applied to arbitrary electrode configurations (Gunther 2004).

I did some tests with different (linear) electrode spreads, different configurations and different spacings, and I noticed that the depth of the parameter mesh is generally set at about 40% of the total spread length. This seems to be confirmed also for a single (e.g. Wenner) quadrupole. 
According to the code I developed to compute the cumulative value of the 1D sensitivity, the depth corresponding to the 90% of the 1D sensitivity does not match the depth automatically set by PyGimli.
So I wander if the software still works as stated in the above paper.

Thank you
Diego

**Comment by halbmy** [comment]
Yes, you are right. When redesigning pyGIMLi to a broad range of methods, this 40% rule was introduced as a general rule for surface data and is not overridden by the ERT manager. This should, however, be the case and will probably be fixed in the future.

**Comment by anonymouscapibara** [comment]
All clear!
Thank you.
